infrastructure/main.tf

```diff
@@ -94,6 +102,19 @@ data "vault_generic_secret" "emGw" {
   path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/em-gw"
 }
 
+# region: for functional/smoke tests
+# todo: create a separate test service just for this app
```

Comment:
Worth adding your test service to this ticket: https://tools.hmcts.net/jira/browse/RDO-1874

Reply:
done